advanced/how_to_debug.rst

```diff
@@ -0,0 +1,101 @@
+.. _how_to_debug:
+
+How to debug
+============
+
+Console and debugger
+--------------------
+
+When you test your game made with Escoria, you can get some errors and warnings
```

Comment:
Please use passive speech here (nothing directed at the developer "you"). For example, this line should be "When testing a game made with Escoria, the console might print errors and warnings".

Reply:
Ah I thought I had all of them removed. Thanks!